worker/download.go

```diff
@@ -339,6 +344,8 @@ outer:
 			if resp.err != nil {
 				mgr.logger.Errorf("download slab %v failed: %v", resp.index, resp.err)
```

Comment:
maybe also print whether we overpaid here. To avoid getting stuck in a loop where we keep failing and overpaying without noticing.